examples/url-shotener/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"github.com/dicedb/dicedb-go" // DiceDB Go SDK
+)
+
+type URL struct {
+	ID       string `json:"id"`
+	LongURL  string `json:"long_url"`
+	ShortURL string `json:"short_url"`
+}
+
+var db *dicedb.Client
+
+// Initialize DiceDB connection
+func init() {
+	db = dicedb.NewClient(&dicedb.Options{
+		Addr: "localhost:7379",
```

Comment:
Get hostname and port from ENV variables?
Similar to: https://github.com/DiceDB/dice/blob/44779c2b0763eecfcf869a79d5936f013e4422e7/examples/leaderboard-go/main.go#L36-L44